src/backend/src/common/swagger.ts

```diff
@@ -23,7 +24,13 @@ export const SwaggerDocs = (app: INestApplication) => {
     .build();
 
   const baseDocument = SwaggerModule.createDocument(app, options, {
-    include: [AppModule, HealthModule, PpqModule, AuthModule],
+    include: [
+      AppModule,
+      HealthModule,
+      PpqModule,
+      AuthModule,
+      ConfigurationModule,
```

Comment:
I realise you just added one thing to the list here, but I think that because `AppModule` imports these other modules we don't need to add them here. You can just add `AppModule`. I _think_...

Reply:
remove it from here, the API endpoint will disappear from swagger

Reply:
oh really? never mind then